components/ws-manager-mk2/controllers/workspace_controller.go

```diff
@@ -500,8 +500,8 @@ func (r *WorkspaceReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		return []string{owner.Name}
 	}
 	err := mgr.GetFieldIndexer().IndexField(context.Background(), &corev1.Pod{}, wsOwnerKey, idx)
-	if err != nil {
-		return err
+	if err != nil && err.Error() == "informer has already started" {
```

Comment:
interesting, when does this happen?

Reply:
This happens when you start/stop controllers during a leader election (this is not required anymore)